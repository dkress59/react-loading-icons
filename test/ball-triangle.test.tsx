import { render } from '@testing-library/react'
import React from 'react'

import BallTriangle from '../src/components/ball-triangle'

describe('<BallTriangle /> component', () => {
	it('renders and matches snapshot', () => {
		const { container } = render(<BallTriangle />)
		expect(container).not.toBeNull()
		expect(container).toMatchSnapshot()
	})
})
