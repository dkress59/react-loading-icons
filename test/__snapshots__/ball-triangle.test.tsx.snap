// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BallTriangle /> component inherits props 1`] = `
<svg
  class="icon-loading test"
  data-testid="icon"
  height="2em"
  viewBox="0 0 57 57"
  width="auto"
>
  <g
    fill="none"
    fill-rule="evenodd"
    stroke-width="2"
    transform="translate(1 1)"
  >
    <circle
      cx="5"
      cy="50"
      fill="#000"
      fill-opacity=".5"
      r="5"
      stroke="#fff"
      stroke-opacity=".5"
    >
      <animate
        attributeName="cy"
        begin="0s"
        calcMode="linear"
        dur="2.2s"
        repeatCount="indefinite"
        values="50;5;50;50"
      />
      <animate
        attributeName="cx"
        begin="0s"
        calcMode="linear"
        dur="2.2s"
        repeatCount="indefinite"
        values="5;27;49;5"
      />
    </circle>
    <circle
      cx="27"
      cy="5"
      fill="#000"
      fill-opacity=".5"
      r="5"
      stroke="#fff"
      stroke-opacity=".5"
    >
      <animate
        attributeName="cy"
        begin="0s"
        calcMode="linear"
        dur="2.2s"
        from="5"
        repeatCount="indefinite"
        to="5"
        values="5;50;50;5"
      />
      <animate
        attributeName="cx"
        begin="0s"
        calcMode="linear"
        dur="2.2s"
        from="27"
        repeatCount="indefinite"
        to="27"
        values="27;49;5;27"
      />
    </circle>
    <circle
      cx="49"
      cy="50"
      fill="#000"
      fill-opacity=".5"
      r="5"
      stroke="#fff"
      stroke-opacity=".5"
    >
      <animate
        attributeName="cy"
        begin="0s"
        calcMode="linear"
        dur="2.2s"
        repeatCount="indefinite"
        values="50;50;5;50"
      />
      <animate
        attributeName="cx"
        begin="0s"
        calcMode="linear"
        dur="2.2s"
        from="49"
        repeatCount="indefinite"
        to="49"
        values="49;5;27;49"
      />
    </circle>
  </g>
</svg>
`;

exports[`<BallTriangle /> component renders and matches snapshot 1`] = `
<svg
  class="icon-loading"
  data-testid="icon"
  height="57"
  viewBox="0 0 57 57"
  width="57"
>
  <g
    fill="none"
    fill-rule="evenodd"
    stroke-width="2"
    transform="translate(1 1)"
  >
    <circle
      cx="5"
      cy="50"
      r="5"
      stroke="#fff"
    >
      <animate
        attributeName="cy"
        begin="0s"
        calcMode="linear"
        dur="2.2s"
        repeatCount="indefinite"
        values="50;5;50;50"
      />
      <animate
        attributeName="cx"
        begin="0s"
        calcMode="linear"
        dur="2.2s"
        repeatCount="indefinite"
        values="5;27;49;5"
      />
    </circle>
    <circle
      cx="27"
      cy="5"
      r="5"
      stroke="#fff"
    >
      <animate
        attributeName="cy"
        begin="0s"
        calcMode="linear"
        dur="2.2s"
        from="5"
        repeatCount="indefinite"
        to="5"
        values="5;50;50;5"
      />
      <animate
        attributeName="cx"
        begin="0s"
        calcMode="linear"
        dur="2.2s"
        from="27"
        repeatCount="indefinite"
        to="27"
        values="27;49;5;27"
      />
    </circle>
    <circle
      cx="49"
      cy="50"
      r="5"
      stroke="#fff"
    >
      <animate
        attributeName="cy"
        begin="0s"
        calcMode="linear"
        dur="2.2s"
        repeatCount="indefinite"
        values="50;50;5;50"
      />
      <animate
        attributeName="cx"
        begin="0s"
        calcMode="linear"
        dur="2.2s"
        from="49"
        repeatCount="indefinite"
        to="49"
        values="49;5;27;49"
      />
    </circle>
  </g>
</svg>
`;
